### DISCUSSION TITLE
Changing Color

### DISCUSSION THREAD

**Discussion by HirayaEs** [twbs/bootstrap#37548] [Q&A] [answered]
how can I change the color of bootstrap into my specific hex code?

**Comment by TianMeds** [comment]
You can't change color to a specific hex code of yours you can only use the classes given in the documentation of bootstrap

**Comment by mdo** [comment, marked as answer]
Because I think you're talking to yourself here with another account, I'll answer this more directly. You can definitely change colors to whatever you want—by changing Sass variables, changing CSS variables, adding custom styles, etc. Plenty of ways to customize every part of Bootstrap, as always.
